### DISCUSSION TITLE
Version reported by wayvnc -V looks wrong

### DISCUSSION THREAD

**Discussion by andrum993** [any1/wayvnc#262] [General]
Currently running wayvnc 0.6.2 on Raspberry Pi OS bookworm beta. Apt reports the version installed is 0.6.2-1~bpo12+1, which seems to be a backport of the debian package of wayvnc 0.6.2. However, running `wayvnc -V` reports this:

```
andrew@bookworm:~ $ wayvnc -V
wayvnc: 0.6.0
neatvnc: 0.6.0
aml: 0.3.0
```

I would have expected it to say 0.6.2 for wayvnc. Why does it show a different version number?

**Comment by any1** [comment]
I forgot to change the version number and doing a new patch release to fix it seems rather excessive.
